core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.html

```diff
@@ -117,31 +117,47 @@
         <div class="e2e-test-add-chapter-outline">
           <div class="story-node-header oppia-mobile-collapsible-card-header" (click)="toggleChapterOutline()">
             <span>Chapter Outline</span>
-            <i class="fa fa-caret-down"
-               *ngIf="!chapterOutlineIsShown"
-               aria-hidden="true">
-            </i>
-            <i class="fa fa-caret-up"
-               *ngIf="chapterOutlineIsShown"
-               aria-hidden="true">
-            </i>
           </div>
-          <div class="chapter-card-inputs oppia-mobile-collapsible-card-content" *ngIf="chapterOutlineIsShown">
+          <br>
+          <div class="outline-Finalized-container" *ngIf="oldOutline">
```

Comment:
Please follow existing conventions for CSS classes -- it should be kebab-case with individual words starting with lowercase letters.

Also, why do you need ngIf="oldOutline" here -- what happens if there's no oldOutline?

Reply:
I fix kebab-case and remove `ngIf="oldOutline"`. Then I replace the second PR video with the video that show this fix behavior on 10/8.
![fix_html1](https://github.com/oppia/oppia/assets/140389350/3088377b-7bbd-45de-a44f-49ee807e97ea)